text/0004-licensing.md

```diff
@@ -0,0 +1,43 @@
+# Licensing
+
+intermezzOS will use “open source” licensing, rather than “free software”
+licensing.
+
+Specifically, the book will be under the CC0, and the code will be under
+MIT/Apache 2.0.
+
+## Details
+
+There are two major forces: open source and free software. There are details
+between the two, but the biggest comes from what ‘freedom’ means. The crux of
+the choice is this:
+
+> Should someone be allowed to create a closed-source version of intermezzOS?
+
+Given the goals of intermezzOS, primarily, that learning resources should be
+available as widely as possible, I believe the answer to this question is ‘yes’.
+
+This is a slightly counter-intuitive conclusion. Wouldn’t forcing it to always
+be open mean that things would be more open? In a certain sense, this might be
+true, but it would restrict what students can do with this material, and that
+makes me uneasy. Furthermore, the boot code is derived from @phil-opp’s, and
+that’s currently licensed under MIT/Apache 2.0.
+
+I personally tend to prefer free software, but am not religious about it. I
+think that this project is significantly different from the projects that
+make me prefer it, and so it’s worth using the more liberal license.
+
+Furthermore, as a Rust project, most of the Rust world is using MIT/Apache 2.0.
+Continuing in that tradition makes sense, and makes it easier to use IP across
+the ecosystem.
+
+## Drawbacks
+
+If someone were to create a proprietary intermezzOS and make piles of cash off
```

Comment:
Does this allow someone to simply go to a publisher and say "please print this github repo" and then earn lots of $$$? That sounds unfair.